package.json

```diff
@@ -17,23 +17,22 @@
   ],
   "scripts": {
     "watch": "npx tsc --watch",
-    "build": "npx tsc",
+    "build": "npx tsdx build",
```

Comment:
AFAIK tsdx is been used for sdk/libraires, not standalone node apps. Maybe it changed?